src/windows/main_window.py

```diff
@@ -2589,8 +2591,7 @@ def FrameExported(self, title_message, start_frame, end_frame, current_frame):
             self.unity_launcher.set_property("progress", current_frame / (end_frame - start_frame))
             self.unity_launcher.set_property("progress_visible", True)
         except Exception:
-            # Just ignore
-            pass
+            log.warning('Failed to notify unity launcher of export progress. Frame: %s' % current_frame)
```

Comment:
```suggestion
            log.debug('Failed to notify unity launcher of export progress. Frame: %s' % current_frame)
```
Especially since this will get logged _every_ frame, if there's a general connectivity problem!